server/form-pages/apply/reasons-for-placement/basic-information/placementDate.test.ts

```diff
@@ -116,29 +116,6 @@ describe('PlacementDate', () => {
         expect(page.errors()).toEqual({})
       })
 
-      it('should return an error if the date is not specified', () => {
```

Comment:
Will we even get to this page if we don't know the release date though?

Reply:
yeah fair point